src/Hl7.Fhir.Specification/Specification/Source/DirectorySource.cs

```diff
@@ -9,8 +9,6 @@
 // [WMR 20171023] TODO
 // - Allow configuration of duplicate canonical url handling strategy
 
-#if NET_FILESYSTEM
```

Comment:
Hoe kan deze weg zijn?  Onder NETSTD11 hoort het dan niet meer te compileren, toch?

Reply:
HL7.Fhir.Specification targets only net45 and netstandard2.0. So no netstandard1.1 anymore. That's why this conditional directive could be deleted.

Reply:
Oh crap, yeah, forgot about that.